### DISCUSSION TITLE
Add a spoiler option to channel settings

### DISCUSSION THREAD

**Discussion by fiveseven808** [Kilvoctu/aiyabot#168] [Ideas]
For certain reasons, I've edited lien 507 of stablecog.py to now read  `file = discord.File(fp=buffer, filename=f'SPOILER_{queue_object.seed}-{count}.png')` which hides the result. It would be great to have this as a parameter while using the /draw command and also setting it as a channel default.

**Comment by Kilvoctu** [comment]
Added it to [projects](https://github.com/users/Kilvoctu/projects/12?pane=issue&itemId=25884470), but I'm not sure when I'll get around to it.
